### PR TITLE
FOGL-1370: Intercepts aiohttp.client_exceptions.ClientPayloadError and handles it less allerting.

### PR DESCRIPTION
**Pull request by sstefano69**

### CODE REVIEW

File: python/foglamp/tasks/north/sending_process.py [hunk 1]
```diff
@@ -512,13 +513,20 @@ def _is_north_valid(self):
...
+
+        converted_data = []
         try:
             # Loads data, +1 as > is needed
             readings = await self._readings.fetch(last_object_id + 1, self._config['blockSize'])
 
             raw_data = readings['rows']
             converted_data = self._transform_in_memory_data_readings(raw_data)
 
+        except aiohttp.client_exceptions.ClientPayloadError as _ex:
```
Comment (ashwinscale) [inline python/foglamp/tasks/north/sending_process.py]: This exception is thrown by the StorageClient. The code shows that the real exception is wrapped by a general Exception.
https://github.com/foglamp/FogLAMP/blob/adbe14e1261d66c64a39bc6a12ab31d4c88e8cb4/python/foglamp/common/storage_client/storage_client.py#L919

Comment (sstefano69) [reply python/foglamp/tasks/north/sending_process.py]: The code I saw in the 1370-repro was : 
        except Exception as ex:
            raise

btw, the new/latest code in Develop is no using anymore the try/catch, so it should be fine.

**Comment by praveen-garg** [comment]
@sstefano69 is this issue still exists? are we able to reproduce on `1370-repro` when rebased with develop?

File: python/foglamp/tasks/north/sending_process.py (hunk 1 above)
Comment (sstefano69) [reply python/foglamp/tasks/north/sending_process.py]: @ashwinscale , is this PR with you ?

**Pull request merged**